examples/quickstart/hello_world.cpp

```diff
@@ -10,13 +10,15 @@
 
 #include <pika/init.hpp>
 
+#include <cstdlib>
 #include <iostream>
 
 int pika_main()
 {
     // Say hello to the world!
     std::cout << "Hello World!\n" << std::flush;
-    return pika::finalize();
+    pika::finalize();
+    return EXIT_SUCCESS;
```

Comment:
To do: other tests and examples need to be updated for this as well. This is a breaking change.

Reply:
Done.